modules/microshift-manifests-override-paths.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-applications.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-manifests-override-paths_{context}"]
+= Override the list of manifest paths
+You can override the list of default paths with the following procedure.
+
+.Procedure
+
+. In the configuration file, set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++`.
++
+[source,terminal]
+----
+manifests:
+    kustomizePaths:
+        - "/opt/alternate/path"
+----
+
+. The values of `kustomizePaths` may be glob patterns.
```

Comment:
I am thinking maybe this should be a note nested under the first step, but I am not sure.

Reply:
Yeah, you could combine the 2 examples and point out that a value can be a single path or glob pattern.